tests/test_gpu_examples.py

```diff
@@ -4353,3 +4445,97 @@ def test_hotswapping_compiled_diffusers_model_does_not_trigger_recompilation(sel
         ranks = 7, 13
         with torch._dynamo.config.patch(error_on_recompile=True):  # raise an error on recompilation
             self.check_hotswap_diffusion(do_hotswap=True, ranks=ranks, alpha_scalings=ranks)
+
+
+class TestLoraWithTrainableTokens:
+    @pytest.mark.single_gpu_tests
+    def test_causal_lm_training_lora_resize_embeddings_trainable_tokens(self):
+        r"""
+        Test LoRA with trainable tokens on a resized embedding matrix
+        """
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            bnb_config = BitsAndBytesConfig(
+                load_in_4bit=True,
+                bnb_4bit_quant_type="nf4",
+                bnb_4bit_compute_dtype=torch.float16,
+                bnb_4bit_quant_storage=torch.float16,
+                bnb_4bit_use_double_quant=True,
+            )
+
+            model = AutoModelForCausalLM.from_pretrained(
+                self.causal_lm_model_id,
```

Comment:
This attribute is undefined, you can use `"facebook/opt-350m"`.

Reply:
That's from being in the wrong test class.